### PR TITLE
Feature/90079 net exchange per neighbour

### PR DESCRIPTION
**Pull request by jonasdmoeller**
## Description

Added aggregation of net exchange between neighboring grid areas (step 1 of BRS-023).

## References

* #0000

### CODE REVIEW

File: source/databricks/geh_stream/aggregation_utils/services/coordinator_service.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def SendResultToCoordinator(self, result):
             response = requests.post(self.coordinatorUrl, data=request_body, headers=headers)
```
Comment (johevemi) [inline source/databricks/geh_stream/aggregation_utils/services/coordinator_service.py:47]: coordinatorUrl is change to coordinator_url

File: source/databricks/aggregation-jobs/aggregation_trigger.py [hunk 2]
```diff
@@ -135,18 +139,19 @@
...
+                                         hourly_production_df.toJSON().collect(),
+                                         flex_consumption_df.toJSON().collect(),
+                                         flex_consumption_with_grid_loss.toJSON().collect(),
+                                         hourly_production_with_system_correction_and_grid_loss.toJSON().collect())
 
 
-ourCoordinatorService = CoordinatorService(args)
-ourCoordinatorService.SendResultToCoordinator(aggregationResults.toJSON())
+coordinator_service = CoordinatorService(args)
+coordinator_service.send_result_to_coordinator(aggregation_results.toJSON())
```
Comment (johevemi) [inline source/databricks/aggregation-jobs/aggregation_trigger.py:157]: toJSON was changed to to_json in AggregationResults

File: source/databricks/geh_stream/aggregation_utils/services/coordinator_service.py (hunk 1 above)
Comment (johevemi) [reply source/databricks/geh_stream/aggregation_utils/services/coordinator_service.py:47]: Is corrected in PR [#64](https://github.com/Energinet-DataHub/geh-aggregations/pull/64)

File: source/databricks/aggregation-jobs/aggregation_trigger.py (hunk 2 above)
Comment (johevemi) [reply source/databricks/aggregation-jobs/aggregation_trigger.py:157]: Is corrected in PR [#64](https://github.com/Energinet-DataHub/geh-aggregations/pull/64)

**Review by kristianschneider — COMMENTED** [review]
✔️

**Pull request merged**